docs/detections/rules-ui-create.asciidoc

```diff
@@ -466,6 +471,8 @@ If the `LIMIT` value and **Max alerts per run** value are different, the rule us
 
 - When writing an aggregating query, use the {ref}/esql-commands.html#esql-stats-by[`STATS...BY`] command with fields that you want to search and filter for after alerts are created. For example, using the `host.name`, `user.name`, `process.name` fields with the `BY` operator of the `STATS...BY` command returns these fields in alert documents, and allows you to search and filter for them from the Alerts table. 
 
+- When configuring alert suppression on a non-aggregating query, avoid sorting alerts by descending `@timestamp` order. Doing so can impact the number of alerts that are suppressed, especially if the number of alerts generated is higher than the maximum number of alerts the rule can create when it runs.
```

Comment:
I would rephrase:
 ```suggestion
- When configuring alert suppression on a non-aggregating query, it is recommended to sort results by ascending `@timestamp` order. It will ensure the number of alerts will be suppressed, especially if the number of alerts generated is higher than the maximum number of alerts the rule can create when it runs.
```

Reply:
Updated in [893ab42](https://github.com/elastic/security-docs/pull/5568/commits/893ab42f59ed03a3488dc16698d940a137cdfda0)